site/src/pages/docs/server-docservice.mdx

```diff
@@ -218,3 +220,33 @@ For gRPC:
       ...
   }
   ```
+
+  Using JavaDoc:
+
+  This only works with method parameters.
+  To enable it, you have to add the armeria annotation processor to your project:
+
+  Groovy configuration:
```

Comment:
Gradle

Reply:
Yes, it is gradle but using groovy xD

the one underneath is also gradle but with kotlin...